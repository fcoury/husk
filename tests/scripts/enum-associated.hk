enum Option {
  None,
  Some(string, int),
}

fn main() {
  let option = Option::Some("Hello", 18);
  match option {
    Option::None => println!("None"),
    Option::Some(name, age) => println!("Name: {} age: {}", name, age),
  }
}

main()
