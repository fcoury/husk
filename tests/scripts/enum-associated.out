Name: Hello age: 18